Makefile

```diff
@@ -4,29 +4,40 @@ CURRENT_UID := $(shell id -u)
 CURRENT_GID := $(shell id -g)
 
 # https://github.com/OpenAPITools/openapi-generator-cli
-# SPEC_URL:="https://developer.equinix.com/sites/default/files/fabric-v4-catalog-fabric_v4_3.yaml"
 SPEC_URL:="https://api.swaggerhub.com/apis/equinix-api/fabric/4.9/swagger.yaml"
 
 SPEC_FETCHED_FILE:=spec.fetched.yaml
 SPEC_PATCHED_FILE:=spec.patched.yaml
-IMAGE=swaggerapi/swagger-codegen-cli-v3:3.0.34
-# IMAGE=openapitools/openapi-generator-cli # to use, change gen: to gen-openapitools
+OPENAPI_CODEGEN_TAG=v6.4.0
+OPENAPI_CODEGEN_IMAGE=openapitools/openapi-generator-cli:${OPENAPI_CODEGEN_TAG} # to use, change gen: to gen-openapitools
+SWAGGER_CODEGEN_IMAGE=swaggerapi/swagger-codegen-cli-v3:3.0.34
 VALIDATE_IMAGE=openapitools/openapi-generator-cli
 
 GIT_ORG=equinix-labs
 GIT_REPO=fabric-go
 PACKAGE_PREFIX=fabric
 PACKAGE_MAJOR=v4
-CRI=docker # nerdctl
+CRI=docker
 
-SWAGGER=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${IMAGE}
-VALIDATE=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local ${VALIDATE_IMAGE}
+DOCKER=${CRI} run --rm -u ${CURRENT_UID}:${CURRENT_GID} -v $(CURDIR):/local
+DOCKER_SWAGGER=${DOCKER} ${SWAGGER_CODEGEN_IMAGE}
+DOCKER_OPENAPI=${DOCKER} ${OPENAPI_CODEGEN_IMAGE}
+
+IMAGE=${SWAGGER_CODEGEN_IMAGE}
 
 GOLANGCI_LINT=golangci-lint
 
-#all: pull fetch patch clean gen mod docs move-other patch-post fmt test stage
```

Comment:
Something to consider while you're in here is providing a `generate` task to encapsulate some of the tasks that are wrapped in `all`.  There's [an example of this in `metal-go`](https://github.com/equinix-labs/metal-go/pull/147); a wrapper task like that could be introduced in all of the generated Equinix SDKs so that we're always running the same tasks in any SDK repo, and any language/generator-specific tasks are hidden inside the wrapper tasks.

Reply:
Love it. Will follow that pattern. 👍